### ISSUE TITLE
关于使用该项目代码预测未标注数据集的方式

### ISSUE THREAD

**Issue by hanyimei** [jjljkjljk/BERT-NER#1]
您好，使用了您的train_bert_crf.py代码我能够在自己的数据集上顺利完成NER任务，训练得到.pt文件的模型参数后，我通过修改train_bert_crf.py中evaluate函数，实现了加载想要的.pt文件进行模型测试，但该测试只能用于已标注的数据集：
test_data = processor.get_test_examples()
test_dataset = NerDataset(test_data)
test_dataloader = DataLoader(test_dataset, batch_size=args.batch_size_eval, shuffle=False,num_workers=args.num_workers)
即使用上述的代码去加载数据，我想请问 若想要使用训练好的模型，去预测未标注数据集时，该如何加载数据呢
